kolibri/core/content/migrations/0014_auto_20181218_1132.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-12-18 19:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('content', '0013_auto_20180919_1142'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='file',
+            name='preset',
+            field=models.CharField(blank=True, choices=[('high_res_video', 'High Resolution'), ('low_res_video', 'Low Resolution'), ('video_thumbnail', 'Thumbnail'), ('video_subtitle', 'Subtitle'), ('video_dependency', 'Video (dependency)'), ('audio', 'Audio'), ('audio_thumbnail', 'Thumbnail'), ('document', 'Document'), ('epub', 'ePub Document'), ('document_thumbnail', 'Thumbnail'), ('exercise', 'Exercise'), ('exercise_thumbnail', 'Thumbnail'), ('exercise_image', 'Exercise Image'), ('exercise_graphie', 'Exercise Graphie'), ('channel_thumbnail', 'Channel Thumbnail'), ('topic_thumbnail', 'Thumbnail'), ('html5_zip', 'HTML5 Zip'), ('html5_dependency', 'HTML5 Dependency (Zip format)'), ('html5_thumbnail', 'HTML5 Thumbnail')], max_length=150),
+        ),
+    ]
```

Comment:
@rtibbles this look right?

Reply:
Yup!

Reply:
Please remember to rename migrations from their original names, i.e. `0014_auto_20181218_1132.py` isn't very verbose :)

Reply:
it's very verbose - it's not very meaningful ;-)

Reply:
I know that the precedence isn't optimal, but we need to pick up from somewhere.

![image](https://user-images.githubusercontent.com/374612/50376680-be180780-0610-11e9-9ed8-ff7ba69550c0.png)